src/components/carbon-provider/carbon-provider.component.tsx

```diff
@@ -20,14 +22,16 @@ export const CarbonProvider = ({
   children,
   theme = mintTheme,
   validationRedesignOptIn = false,
-}: CarbonProviderProps) => (
-  <ThemeProvider theme={theme}>
-    <CarbonScopedTokensProvider>
-      <NewValidationContext.Provider value={{ validationRedesignOptIn }}>
-        {children}
-      </NewValidationContext.Provider>
-    </CarbonScopedTokensProvider>
-  </ThemeProvider>
-);
+}: CarbonProviderProps) => {
```

Comment:
nope, definitely not. Thanks for catching this, it's only because I initially put the TopModalContext stuff in there before pulling it out to its own component - at that point it obviously needed the longhand function body but I forgot to change this back when I refactored.